docs/user/reference/charm/index.md

```diff
@@ -97,7 +105,7 @@ Examples:
 (principal-charm)=
 #### Principal
 
-In {ref}`machine charms <machine-charm>`, a **principal charm** is any that is not {ref}`subordinate <subordinate-charm>`.
+In {ref}`machine charms <machine-charm>`, a **principal charm** is any charm that is not {ref}`subordinate <subordinate-charm>`.
```

Comment:
funny. I'd say a principal is a charm that has a subordinate.
It's always "the principal to this subordinate charm", never "this principal charm", or else every charm that isn't a subordinate is a principal because it can have a subordinate.
I'd say a charm is a principal not if it **can have** a subordinate, but when **it actually does**.

But this is very much IMHO-semanticky.

Reply:
Updated to " a **principal charm** is any charm that has a subordinate".